src/_guides/bump-sh-tutorials/api-linting-with-vacuum.md

```diff
@@ -0,0 +1,163 @@
+
+Run vacuum on any OpenAPI available, or download the [Train Travel API](https://github.com/bump-sh-examples/train-travel-api) to use that OpenAPI.
+
+```bash
+npx vacuum lint -d openapi.yaml
+```
+
+The output can get quite large, as vacuum comes with a lot of useful rules baked in (so that every user doesn't have to invent them all by themselves every time.)
+
+![](/images/guides/api-linting-with-vacuum/cli-output-warnings.png)
+
+### Step 3: Configure rules/rulesets
+
+If there is anything fundamentally broken with the API description then vacuum will consider that an "Error", but in this instance it looks like the Train Travel API is missing quite a few `examples`. Maybe that's not something I'm too worried about if I'm not using this OpenAPI document for API documentation or [mock servers](_guides/bump-sh-tutorials/mocking-with-microcks.md), and if that's the case for you you can turn off that rule.
+
+To configure vacuum, create a `.vacuum.conf.yaml` file in your working directory (ideally the root of a Git repository).
+
+```yaml
+# .vacuum.conf.yaml
+ruleset: ./.vacuum/ruleset.yaml
+```
+
+Now let's go make that `./.vacuum` directory file and pop a new file in it.
+
+```yaml
+# .vacuum/ruleset.yaml
+extends: 
+  - spectral:oas
+
+rules:
+  oas3-missing-example: off
+```
+
+By default we're using the Spectral "OAS" (OpenAPI Specification) ruleset, which is where all these rules are defined. Creating this config file and new ruleset is overriding the default functionality, but doing something very similar. We're extending the ruleset so we have all the rules, then turning off only the `oas3-missing-example` rule. 
+
+Running the lint command again should be a little cleaner.
+
+![](/images/guides/api-linting-with-vacuum/cli-output-fewer-problems.png)
+
+If you spot an error in here you would like to fix, you can use the `Location` to find out exactly where it is. `openapi.yaml:1049:5` means it's in `openapi.yaml` on line `1049` and it's taken a stab at guessing which character of the problem, which in this case is 4 spaces of indent plus where is the thing in the 5th character... no worries, close enough.
+
+![](/images/guides/api-linting-with-vacuum/go-to-line.png)
+
+The problem here was a lack of a description, so pop something in like `description: The link to the booking resource.` and run it again to see if that fixed the error. 
+
+Once everything is fixed or ignored, you should get a perfect run.
+
+![](/images/guides/api-linting-with-vacuum/perfect-score.png)
+
+### Step 4: Keep that perfect score
+
+Once everything is fixed, wouldn't it be nice to keep it that way? 
+
+Adding a new GitHub Action can help us do this, and we don't need to get things mixed up with existing GitHub Actions you may have for deploying Bump.sh API documentation, updating mock servers, or generating SDKs, it can just be its own new `lint.yml` workflow.
+
+```yaml
+# .github/workflows/lint.yml
+name: Lint
+
+"on":
```

Comment:
```suggestion
on:
```

For consistency with what's usually seen in the wild, I'd recommend to remove the quotes here :)